proto/prysm/v1alpha1/beacon_state.pb.go

```diff
@@ -1073,7 +1073,7 @@ func (x *SyncAggregatorSelectionData) GetSubcommitteeIndex() uint64 {
 	return 0
 }
 
-type BeaconStateMerge struct {
+type BeaconStateBellatrix struct {
```

Comment:
Renaming in the generated code is not ideal. Please re-run `./hack/update-go-pbs.sh` after modifying proto files.

The reason is that `file_proto_prysm_v1alpha1_beacon_state_proto_rawDesc` attribute should be updated as well and it's some hex bytes so rebuilding the pb.go files is really the only way to do it. You may need to run gofmt or goimports to sort the imports for the linter.

Reply:
ok

Reply:
Done, many files regenerated and pushed